src/System.Diagnostics.Tools/ref/System.Diagnostics.Tools.cs

```diff
@@ -18,7 +18,7 @@ public sealed partial class GeneratedCodeAttribute : System.Attribute
 }
 namespace System.Diagnostics.CodeAnalysis
 {
-    [System.AttributeUsageAttribute((System.AttributeTargets)748, Inherited=false, AllowMultiple=false)]
+    [System.AttributeUsageAttribute((System.AttributeTargets)749, Inherited=false, AllowMultiple=false)]
```

Comment:
These are ints rather than orred flags because of a "bug" in the genapis tool that originally created them. It's fine to leave ints but orred flags are nicer.